test/server/meta.integration.test.js

```diff
@@ -0,0 +1,130 @@
+            'data-uid':                    'uid_gwakesmdhfvakwylnuczrbplzdthxe'
+        }
+    } ],
+    [ {
+        'url':         'https://www.paypal.com/sdk/js?',
+        'client-id':   'AfjD8CKlCiagY3OXl46F4HJKHIP7k2wRU3UDfMSuo52DBiXuc0srow-uyWGeHT0EXd67zADzzkm-5eoy',
+        'components':  'buttons',
+        'currency':    'GBP',
+        'intent':      'authorize',
+        'locale':      'en_GB',
+        'merchant-id': 'ASCDJ8AE36AYQ'
+    }, {
+        'attrs': {
+            'data-amount': '7.99',
+            'data-uid':    'uid_issqwuttcumomcdjzbomwiprpnsgjx'
+        }
+    } ],
+    [ {
+        'url':         'https://www.paypal.com/sdk/js?',
+        'client-id':   'AebICI1y5FXWLTk3MHJ-I8g9_lUMlpxf6AP95fnbdtx3WQQFKpMFBP0KsEzeYQAE4GeQ18DlVxvw1qQ9',
+        'commit':      'false',
+        'currency':    'USD',
+        'components':  'buttons,funding-eligibility',
+        'merchant-id': 'VWPTBSYGC33KJ'
+    }, {
+        'attrs': {
+            'data-partner-attribution-id': 'WIX_SP_EC',
+            'data-uid':                    'uid_gsgwcqyaamrhxpucosxljrmoppaadh'
+        }
+    } ],
+    [ {
+        'url':         'https://www.paypal.com/sdk/js?',
+        'client-id':   'AebICI1y5FXWLTk3MHJ-I8g9_lUMlpxf6AP95fnbdtx3WQQFKpMFBP0KsEzeYQAE4GeQ18DlVxvw1qQ9',
+        'commit':      'false',
+        'currency':    'EUR',
+        'components':  'buttons,funding-eligibility',
+        'merchant-id': 'HG62CLUW5KYW2'
+    }, {
+        'attrs': {
+            'data-partner-attribution-id': 'WIX_SP_EC',
+            'data-uid':                    'uid_gsgwcqyaamrhxpucosxljrmoppaadh'
+        }
+    } ],
+    [ {
+        'url':         'https://www.paypal.com/sdk/js?',
+        'client-id':   'AfjD8CKlCiagY3OXl46F4HJKHIP7k2wRU3UDfMSuo52DBiXuc0srow-uyWGeHT0EXd67zADzzkm-5eoy',
+        'components':  'buttons',
+        'currency':    'GBP',
+        'intent':      'authorize',
+        'locale':      'en_GB',
+        'merchant-id': 'ASCDJ8AE36AYQ'
+    }, {
+        'attrs': {
+            'data-amount': '57.00',
+            'data-uid':    'uid_ssqwuttcumomcdjzbomwijromscjzs'
+        }
+    } ],
+    [ {
+        'url':         'https://www.paypal.com/sdk/js?',
+        'client-id':   'AfjD8CKlCiagY3OXl46F4HJKHIP7k2wRU3UDfMSuo52DBiXuc0srow-uyWGeHT0EXd67zADzzkm-5eoy',
+        'components':  'buttons',
+        'currency':    'EUR',
+        'intent':      'authorize',
+        'locale':      'de_DE',
+        'merchant-id': '3PA5M5QDS5BES'
+        
+    }, {
+        'attrs': {
+            'data-amount': '29.90',
+            'data-uid':    'uid_ssqwuttcumomcdjzbomwiakaomcjzs'
+        }
+    } ]
+];
+
+/**
+ * Generate the URL from the source object
+ *
+ * @param {object} sourceData to generate the URL
+ * @returns {string} the generated URL including all the query parameters
+ */
+function buildUrl(sourceData) : string {
+    const { url, ...queryParams } = sourceData;
+    const stringQueryParams = Object
```

Comment:
Can you leverage `URLSearchParams` to simplify converting an object to a querystring?
<img width="560" alt="Screen Shot 2022-02-17 at 2 27 55 PM" src="https://user-images.githubusercontent.com/534034/154565306-aabc0dd1-44ed-4523-acde-85306fbd062f.png">

Reply:
Sure @gregjopa, I'll make the changes